ApollosProject/apollos-prototype
# Adds ActionList on the Home feed driven by a Rock content channel.

## DESCRIPTION

Blocked by #828 

### What does this PR do, or why is it needed?

Takes all the API work I've done, and applies it to the front end. Now the ActionLists on the home feed are driven by a persona feed and a rock content channel. This is all configurable via the API.

Closes #800 
### What design trade-offs/decisions were made?

In a future PR, I will take the `ContentTableCard` component and move it to the UI-kit. 

### How do I test this PR?

1. Open the app.
2. Check out the sweet new content on the home feed.
- Click on them and make sure they open up content. 

### What automated tests did you write?

n/a

## TODO:

- [x] Closes #800 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
